kostyor/conf.ini

```diff
@@ -1,3 +1,4 @@
 [global]
 host = 127.0.0.1
 port = 5000
+discovery_methods = first,second
```

Comment:
Dima, why did you add 'first' and 'second' discovery methods? Why didn't you add 'openstack'  method, which is already implemented in inventory/discover.py?